### DISCUSSION TITLE
Creatable on empty select

### DISCUSSION THREAD

**Discussion by ricick** [mantinedev/mantine#2466] [Q&A] [answered]
Setting creatable on a Select with no options doesn't seem to work. Am I missing something here? The dropdown containing the text in the search box and the create prompts never shows due to there being no options from what I can see.

**Comment by ricick** [comment]
Multiselect doesn't seem to have this problem.

**Comment by rtivital** [comment, marked as answer]
@ricick can you provide a sandbox with a reproduction?
@achmurali can you investigate this issue?

**Reply by ricick** [reply]
Hmm vanilla example in codesandbox seems to work fine so it must be something about my implementation. Will keep digging for now.

**Reply by ricick** [reply]
all good, I was mistaken.
